src/scan.l

```diff
@@ -233,6 +233,8 @@ D	[0-9]+
 HEX	[0-9a-fA-F]+
 IDCOMPONENT [A-Za-z_][A-Za-z_0-9]*
 ID	{IDCOMPONENT}(::{IDCOMPONENT})*
+/* Token including including :: prefix for lookups of global identifiers */
+GLOBAL_ID	::{IDCOMPONENT}
```

Comment:
I don't think `GLOBAL_ID` is a great name, but didn't quite come up with anything else. Suggestions welcome.